lib/plugins/haskell.js

```diff
@@ -10,6 +11,7 @@ export default class Haskell {
       `{BASE_URL}/${user}/${repo}/blob/master/src/${filePath}.hs`,
       `{BASE_URL}/${user}/${repo}/blob/master/lib/${filePath}.hs`,
       `{BASE_URL}/${user}/${repo}/blob/master/${filePath}.hs`,
+      githubSearch({ path, target: `${filePath}.hs` }),
```

Comment:
@ousennin I updated the haskell plugin in order to improve the coverage. This new resolver will perform a github search on the current repo in order to find the linked file. 

I didn't removed the other path definitions, since they are still valid in many cases and more accurate.